test/release_notes.py

```diff
@@ -0,0 +1,177 @@
+import dataclasses
+import hashlib
+import os
+import tempfile
+
+import yaml
+
+import ocm
+import concourse.steps.release as csr
+import release_notes.model as rnm
+import release_notes.tarutil as rnt
+
+
+class DummyOciClient:
```

Comment:
[magic-mock](https://docs.python.org/3/library/unittest.mock.html) might be useful..